TestPlaybooks/playbook-Extract_Indicators_From_File_-_test_4_0.yml

```diff
@@ -275,7 +275,7 @@ tasks:
         simple: "yes"
       unsecure: {}
       url:
-        simple: https://raw.githubusercontent.com/demisto/content/master/TestData/docxwithindicators.docx
+        simple: https://raw.githubusercontent.com/demisto/content/master/TestData/docxwithindicators
```

Comment:
Seem like a bug. I think it should be with `.doc` ending

Reply:
Done by mistake. see https://github.com/demisto/content/pull/3845#discussion_r303747258